### PR TITLE
Set up base JSON API

### LINKED ISSUE #8: Set up base JSON API
## Why

To set up base JSON API structure 

## Acceptance Criteria
- [ ]  Set up base JSON API
- [ ]  Set up JSON API serializer
- [ ] Added unit test

### PR DESCRIPTION
**Pull request by junan**
https://github.com/junan/google-scraper/projects/1#card-50542004

## What happened 👀

Set up base JSON API

## Insight 📝

- Used [jsonapi](https://github.com/google/jsonapi) serializer
- Used Beego's default json rendering with `ServeJSON `
- Added `api/v1/me` api(Later will refactor once I have oauth authentication)

## Proof Of Work 📹

Screen Shot 2021-03-29 at 12 10 51 PM

### CODE REVIEW

File: controllers/api/v1/me_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	"net/http"
+
+	. "google-scraper/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("MeController", func() {
+	Describe("GET /api/v1/me", func() {
```
Comment (olivierobert) [inline controllers/api/v1/me_test.go]: How about creating a health check endpoint [as done by Hoang](https://github.com/hoangmirs/go-scraper/pull/54/files#diff-f274235a2d89036f75c19f1bea39fb17df98bae043b6f46815494a81edf4da49). At least, it's not something you will have to remove later and it's actually useful.

Comment (junan) [reply controllers/api/v1/me_test.go]: Updated in https://github.com/junan/google-scraper/pull/45/commits/82c0c77bf3b8e8211485ad61b3cc7ab48a23d79c

**Review by olivierobert — APPROVED** [review]

File: controllers/api/v1/health_check_test.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	Describe("GET /api/v1/health-check", func() {
+		It("returns 200 status code", func() {
+			response := MakeRequest("GET", "/api/v1/health-check", nil)
+
+			Expect(response.StatusCode).To(Equal(http.StatusOK))
+		})
+
+		It("returns correct JSON response", func() {
+			expectedResponse := `{
+				"data": {
```
Comment (Lahphim) [inline controllers/api/v1/health_check_test.go:23]: @junan Can you check with Hoang's PR?
He did with checking valid json scheme

Comment (junan) [reply controllers/api/v1/health_check_test.go:23]: I have seen his PR where he used schema matcher. But I prefer to use Gomega's `MatchJSON` matcher  😄

**Review by Lahphim — APPROVED** [review]

**Review by gutakk — APPROVED** [review]

**Pull request merged**